iphone/Maps/LocalizedStrings/fa.lproj/Localizable.strings

```diff
@@ -1251,6 +1251,9 @@
 /* Outdoors/hiking map style (activity) name in the Styles and Layers dialog */
 "button_layer_outdoor" = "پیاده روی";
 
+/* Bookmark categories screen, button that opens share dialog to export all bookmarks and tracks */
+"bookmarks_export" = "همه نشانکu200cها و آهنگu200cها را صادر کنید";
```

Comment:
u200c - is it a bug? How does it look on the screen?

Reply:
Yes, this is bug in the `strings`... 
I'll try to fix that
<img width="350" alt="image" src="https://github.com/organicmaps/organicmaps/assets/79797627/e6c5a2e0-4b86-4fad-a216-d9982f0985f5">

Reply:
DeepL doesn't have Fathi (Persian). The python script internally uses Google Translate for languages that are not supported by DeepL. Check the script output.